server/api.js

```diff
@@ -812,15 +813,16 @@ router.deleteAsync("/lobby-referee", ensure.isRef, async (req, res) => {
  * Params:
  *  - lobby: the _id of the lobby
  *  - teams: true to add team, false to add player
+ *  - user: name of the person to add (default: self)
  *  - tourney: identifier of the tournament
  */
 router.postAsync("/lobby-player", ensure.loggedIn, async (req, res) => {
-  if (!req.user.tournies.includes(req.body.tourney)) return res.status(403).send({});
-  logger.info(`${req.user.username} signed up for a quals lobby in ${req.body.tourney}`);
+  if (!req.body.user && !req.user.tournies.includes(req.body.tourney)) return res.status(403).send({});
```

Comment:
with this, any logged in user could sign anyone else up for a lobby. it might be good to validate that the user `isAdmin` before allowing a custom `req.body.user`

Reply:
done